hazelcast/src/main/java/com/hazelcast/instance/MobyNames.java

```diff
@@ -0,0 +1,1007 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
```

Comment:
I am not sure but I think we add a line like `Copyright ... Moby` here when copying code.

Reply:
Fixed.